frontends/mit-learn/src/pages/ChannelPage/ChannelPage.test.tsx

```diff
@@ -247,6 +262,39 @@ describe.each(NON_UNIT_CHANNEL_TYPES)(
         ])
       })
     })
+
+    if (channelType === ChannelTypeEnum.Topic) {
```

Comment:
Maybe we should sense to add a new describe block? `describe("Topic Channel unique behavior", () => {...})`

(and / or split these out into separate files)

Reply:
I moved the test to its own `describe` block, but:

```ts
      if (
        channel.channel_type === ChannelTypeEnum.Topic &&
        channel.topic_detail.topic
      ) {
```

...is still necessary because otherwise Typescript will complain about the properties not existing, even though I'm explicitly passing `ChannelTypeEnum.Topic` to `setupApis`. It can't introspect that a topic channel will only be returned with that argument passed.